llvm_passes/HipDynMem.cpp

```diff
@@ -268,6 +266,14 @@ class HipDynMemExternReplacePass : public ModulePass {
         }
       }
     }
+#else
+    SmallVector<Constant *, 8> Cnst;
+    for (GlobalVariable *GV : GVars) {
+      Constant *CE = cast<Constant>(GV);
+      Cnst.push_back(CE);
+    }
+    convertUsersOfConstantsToInstructions(Cnst);
```

Comment:
I think, the `GVars` might be passed directly with:
```
convertUsersOfConstantsToInstructions(ArrayRef(GVars));
```

Reply:
Does not work:

```
/home/michal/0/source/chip-spv/llvm_passes/HipDynMem.cpp:270:5: error: no matching function for call to 'convertUsersOfConstantsToInstructions'
  270 |     convertUsersOfConstantsToInstructions(ArrayRef(GVars));
      |     ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/home/LLVM_17/include/llvm/IR/ReplaceConstant.h:27:6: note: candidate function not viable: no known conversion from 'ArrayRef<llvm::GlobalVariable *>' to 'ArrayRef<Constant *>' for 1st argument
```

Reply:
Fair enough.